### DISCUSSION TITLE
Obsidian Text Extractor (Supports Omnisearch)

### DISCUSSION THREAD

**Discussion by jwafu** [pieces-app/opensource#16] [Contribution Ideas]
[Link to the Obsidian Text Extractor Github Repo](https://github.com/scambier/obsidian-text-extractor)

Scambier currently is running out of time to dedicate to this plugin, and it is depended on by a lot by Omnisearch, which is a fan favorite extension in obsidian - if you have not used it i recommend giving it a try.

Would love to start some conversations around how we can assist here and start lending a hand. Eventually we could also see if/how the pieces-os-client can benefit the logic they currently have around code, or some of our other OCR features.

You can test our drag and drop OCR in the app for sure!

**Comment by JeremiahBoothe** [comment]
I'd love to be involved but I'm not that good of a programmer yet, I'm just finally finishing my degree at 43, heading into my senior year after Christmas, but most of my programming experience has been random on and off as a hobbyist since like '96.  My strongest languages are Java and Kotlin, with C++ being next after that, I don't know TypeScript or JavaScript but I do hope it all works out.  

When I tried pieces for the first time and saw the potential the whole AI craze suddenly made sense to me.  I've used Obsidian for over a year and primarily use JetBrains IDE so having that cross integration and it suddenly felt like the Obsidian promise of second brain was coming true.

**Reply by jwafu** [reply]
@JeremiahBoothe honestly i loveeeee obsidian its been so great for me  - markdown is just better. the two of us could collaborate on this here - we could collaborate on some issues and get you more familiar in the environment, even just as an obsidian user it would be helpful to have more insights and testers while working on trying to contribute here. 

I actually could use your assistance on a build.gradle.kts file if you have some time over the weekend/know anything about that there - but regardless i think there is an opportunity here where we could use your help on one of these fronts

**Reply by JeremiahBoothe** [reply]
Awesome! I sent you a message on discord.

**Comment by jwafu** [comment]
@JeremiahBoothe for later this week here is a great first issue im planning on looking into probably towards friday or saturday if not sooner
